jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -52,6 +52,8 @@
 import jdk.internal.misc.SharedSecrets;
 /*[ENDIF] JAVA_SPEC_VERSION >= 17 */
 
+import com.ibm.oti.vm.VM;
```

Comment:
Please move this up after `package openj9.internal.criu;` to keep the imports in order.